docs/pvcscale.md

```diff
@@ -0,0 +1,221 @@
+
+PVCScale test creates a desired number of Pods and attaches one PVC to each Pod, it supports 
+only dynamic PVC allocation using storage class and in order for this test to 
+work it is a requirement to have a working storage class which can be used to allocate a PVC.
+
+At end of test, the time necessary for Pods to reach `READY` state with PVC
+in `Bound` status is measured, and based on this time we can compare start time
+for same number of Pods/PVCS for cases when only difference is storage class
+used to allocate PVCS. 
+
+The situation when Pod is in `READY` state and  with PVC in `Bound` status is considered as state where PVC is
+ready for consumption by application Pod.
+
+Different storage backends behaves differently when it comes to PVC scalability and it is necessary to have 
+repeatable results which can answer the question what storage backend provides optimal PVCs scale and density results for
+scenario where many Pods/PVCs are necessary to start in short period of time.
+
+Running from CLI:
+
+```sh
+$ cp workloads/inventory.example inventory
+$ # Add orchestration host to inventory
+$ # Edit vars in workloads/vars/pvcscale.yml or define Environment vars (See below)
+$ time ansible-playbook -vv -i inventory workloads/pvcscale.yml
+```
+
+## Environment variables
+
+### PUBLIC_KEY
+Default: `~/.ssh/id_rsa.pub`
+
+Public ssh key file for Ansible.
+
+### PRIVATE_KEY
+Default: `~/.ssh/id_rsa`
+
+Private ssh key file for Ansible.
+
+### ORCHESTRATION_USER
+Default: `root`
+
+User for Ansible to log in as. Must authenticate with PUBLIC_KEY/PRIVATE_KEY.
+
+### WORKLOAD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`
+
+Container image that runs the workload script.
+
+### WORKLOAD_JOB_NODE_SELECTOR
+Default: `true`
+
+Enables/disables the node selector that places the workload job on the `workload` node.
+
+### WORKLOAD_JOB_TAINT
+Default: `true`
+
+Enables/disables the toleration on the workload job to permit the `workload` taint.
+
+### WORKLOAD_JOB_PRIVILEGED
+Default: `true`
+
+Enables/disables running the workload Pod as privileged.
+
+### KUBECONFIG_FILE
+Default: `~/.kube/config`
+
+Location of kubeconfig on orchestration host. PVCScale tests supports only
+default location for `kubeconfig` file.
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
+Default: `~/.ssh/id_rsa`
+
+Location of ssh private key to authenticate to the pbench results server.
+
+### PBENCH_SSH_PUBLIC_KEY_FILE
+Default: `~/.ssh/id_rsa.pub`
+
+Location of the ssh public key to authenticate to the pbench results server.
+
+### ENABLE_PBENCH_AGENTS
+Default: `false`
+
+Enables/disables the collection of pbench data on the pbench agent Pods. These Pods are deployed by the workloads/tooling.yml playbook
+it needs to be executed independently of PVCScale test.
+
+### PBENCH_SERVER
+Default: `There is no public default.`
+
+DNS address of the pbench results server.
+
+### SCALE_CI_RESULTS_TOKEN
+Default: `There is no public default.`
```

Comment:
Same as above ^, Since this is no real value I think the backticks surrounding the quote "There is no public default." should be removed.

Reply:
done